core/rawdb/chain_iterator.go

```diff
@@ -28,157 +28,164 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethdb"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rlp"
+	"golang.org/x/crypto/sha3"
```

Comment:
Why use `golang.org/x/crypto/sha3` here?

Reply:
The hasher-processors re-use their hashers, which are `sha3.NewLegacyKeccak256`